packages/shared/src/storage/GoogleDriveStorage/GoogleDriveStorage.android.ts

```
@@ -0,0 +1,265 @@
+    await RNCloudFs.loginIfNeeded();
+
+    return RNCloudFs.deleteFromCloud({ id: fileId });
+  }
+
+  async listFiles(): Promise<{ files: IGoogleDriveFile[] }> {
+    // Ensure user is signed in
+    const signedIn = await GoogleSignin.isSignedIn();
+    if (!signedIn) {
+      throw new OneKeyLocalError(
+        'Not signed in to Google. Please sign in first.',
+      );
```

ESLint

[error] Expected an error object to be thrown.

(@typescript-eslint/no-throw-literal)